src/objects/call_back_query.rs

```diff
@@ -0,0 +1,14 @@
+use objects::message::Message;
+use objects::user::User;
+
+/// represents an incoming callback query from a callback button in an inline keyboard.
```

Comment:
Please use capital letters at the beginning of a sentence. When referring to another type, like "inline keyboard", use the type like `InlineKeyboardMarkup`.